### ISSUE TITLE
Page Load Error - Droid Turbo 2

### ISSUE THREAD

**Issue by geeko55** [cluracan/pocketfurc#29]
For the last week or so, any attempt to use the app is met with a Page Load Error. Screen initializes once, flashes black, then displays the plain text message Page Load Error. I'm on the latest system software, 25.201.6

Any help with this would be great, as otherwise I really do like the app.

**Comment by merklejerk** [comment]
I can't seem to reproduce this on any of my devices. Was there a system update, or possibly a chrome update, immediately prior to this happening?

**Comment by geeko55** [comment]
I don't believe so. I went ahead and updated to the most recent version of Chrome available, 59.0.3071.117, then reinstalled Pocket Furc, then power cycled, but I'm still having issues. Once I'm done with work today I will see if I can dig through the app code and try and find the issue as well

**Comment by geeko55** [comment]
Ah, just thought I should clarify: it's Page load failed, not error. My mistake. Further, I've tried clearing both Chrome and Pocket Furc's cache, as well as my system cache, to no help.

**Comment by phoe** [comment]
A friend of mine confirms this issue.
Samsung Galaxy J3, Model Number SM-J320FN, Android version 5.1.1, Chrome version 58.0.3029.83

**Comment by englebertbigbic** [comment]
Yesterday app worked fine, Last night google updated lots of things including the Chrome browser and now this morning it no longer works. Just gives "Page load error" In top left of screen.

**Comment by Adek713** [comment]
I've had the same issue for about a week too. On both my Tab S2 and Galaxy S8+, both updated to the latest android.

**Comment by merklejerk** [comment]
Finally started receiving this error now as well. Guessing it's due to a chrome webview content policy update. I'll try to figure it out and patched this week.

**Comment by merklejerk** [comment]
Well, it seems like recent changes to the android sdk have broken the android framework pocketfurc uses ([Mobile Chrome Apps](https://github.com/MobileChromeApps/mobile-chrome-apps)). That framework is no longer in active development since chrome apps are being phased out. Since pocketfurc has been living on borrowed time for the past year or so, the proper solution is to disentangle pocketfurc from chrome then port it to cordova for mobile and electron for desktop. This isn't trivial at all so I've unpublished the app from the play store until this can all get sorted out. No ETA on when pocketfurc will return, sorry.

**Comment by englebertbigbic** [comment]
Was wondering if there was an update on how the unfortunate overhaul is going?

**Comment by NisiDiamondfire** [comment]
Any update? I just got a Galaxy Fiesta and went to open pocketfurc to meet the page load error message. Made me so sad...

**Comment by merklejerk** [comment]
Sorry, I've really got no time to devote to this project right now. I was hoping the web client would be out by now to pick up the slack.

**Comment by NisiDiamondfire** [comment]
Well that makes me sad.... It was a great app. Hope you are doing well.
Guess I'll just have to wait then
